README.md

```diff
@@ -1 +1,46 @@
-##Network Service
+# Network Service 
+
+Network layer for creating different set of network requests like GET, POST, PUT, DELETE 
+⚠️ The network layer is under active development.
+
+## Current dev environment 
+
+| Dev environment| Ref link |
+| --- | -- |
+| ![macOS](https://img.shields.io/badge/macOS-13.3+-blue) | | 
+| ![Xcode](https://img.shields.io/badge/Xcode-14.3-red) | |
```

Comment:
Can we use `Xcode 15.0`?

Reply:
we freeze our dev environment, but users could use 15.0

Reply:
So tell that.
Put this information into the README and/or update the badges so everyone knows what `Xcode` version is valid to use.

Reply:
Have you tried to build the package with `Xcode 15`?
Have you tried to install the package using the `Xcode 15`?

Reply:
Yes, these items are tested on Xcode 15 beta